src/main/java/rx/internal/util/ObjectPool.java

```diff
@@ -54,11 +49,11 @@ private ObjectPool(final int min, final int max, final long validationInterval)
         // initialize pool
         initialize(min);
 
-        schedulerWorker = Schedulers.computation().createWorker();
```

Comment:
I wanted to remove interference from the computation scheduler threads but I haven't noticed any effect on benchmark running less than 67 seconds. I'll remove this change.